tests/ssl-opt.sh

```diff
@@ -525,6 +525,46 @@ check_server_hello_time() {
     fi
 }
 
+# Get handshake memory usage from server or client output and check if this value 
+# is not higher then maximum given in the argument
+handshake_memory_check() {
+    MAX_MEMORY="$1"
+    OUTPUT_FILE="$2"
+
+    # Get memory usage from patern like "Heap memory usage after handshake: 23112 bytes. In peak was 33112"
+    MEMORY_USAGE=$(sed -n 's/.*Heap memory usage after handshake: //p' < "$OUTPUT_FILE" | grep -o "[0-9]*" | head -1)
```

Comment:
Instead of duplicating this, could we call `handshake_memory_get()`?

Reply:
In C language it would be much better but in this case it looks more readable in my opinion

Reply:
I think @Patater might have suggested removing the duplication because of the ease of maintenance and not readability. If we have only a single instance of this expression, we need to make changes at a single location whenever we need to update it. This means less work and less risk that they get out of sync.

Reply:
I changed it